frontend-react/src/pages/deliveries/Table/ReportLink.tsx

```diff
@@ -62,11 +62,14 @@ function ReportLink(props: Props) {
 
     if (!props.button) {
         return (
-            <a href="/" onClick={handleClick} className="usa-link">
+            <button
+                onClick={handleClick}
```

Comment:
As a general rule, no link should have `onClick` behavior other than navigating to a page. This will always be a case for a `button`